src/core/namespace.c

```diff
@@ -1268,8 +1293,8 @@ static int mount_image(
                                        host_os_release_id,
                                        host_os_release_version_id ? " VERSION_ID=" : "",
                                        strempty(host_os_release_version_id),
-                                       host_os_release_sysext_level ? " SYSEXT_LEVEL=" : "",
-                                       strempty(host_os_release_sysext_level));
+                                       host_os_release_level ? image_class_info[class].level_env : "",
```

Comment:
`=` is now missing from the output so it will look wrong, it needs to be printed